src/core/profiles/dto/responses.dto.ts

```diff
@@ -121,3 +122,48 @@ export class ResponsePaginationProfile {
   })
   pagination: PaginationMessage;
 }
+
+export class ProfileData {
+  @ApiProperty()
+  id: number;
+  @ApiProperty()
+  userId: number;
+  @ApiProperty()
+  description: string;
+  @ApiProperty()
+  mainTitle: string;
+  @ApiProperty()
+  countryResidence: string;
+  @ApiProperty()
+  deletedAt: Date;
+  @ApiProperty({
+    type: [ExperienceData],
+  })
+  experience: Experience[];
+  @ApiProperty({
+    type: [SkillData],
+  })
+  skills: Skill[];
```

Comment:
El tipo `SkillData` es distinto de `Skill` especificado mas abajo, ¿por que?
```suggestion
  @ApiProperty({
    type: [SkillData],
  })
  skills: Skill[];
```

Reply:
swagger se chispotea cuando una entidad tiene muchas relaciones y no puede hacer el ejem response asi que te recomienda hacer una clase para que la respuesta example funcione

Reply:
Tenemos que revisar esto juntos despues